common/src/main/java/me/shedaniel/architectury/event/events/GuiEvent.java

```diff
@@ -32,62 +32,131 @@
 
 import java.util.List;
 
+/**
+ * Should be moved to the client package and renamed to ScreenEvent in version 2.0
```

Comment:
There should be a more descriptive opening line for the javadoc than this.

Reply:
It is just for internal purpose, is there a annotation I can use or should I just add a todo so we won't forget about it?

Reply:
Not sure if this should be renamed to ScreenEvent, since this is much more than only Screens.